components/service/fretboard/src/main/java/mozilla/components/service/fretboard/source/kinto/KintoExperimentSource.kt

```diff
@@ -18,14 +20,14 @@ import org.json.JSONObject
  * @property baseUrl Kinto server url
  * @property bucketName name of the bucket to fetch
  * @property collectionName name of the collection to fetch
- * @property client http client to use
+ * @param client http client to use
  */
 class KintoExperimentSource(
     private val baseUrl: String,
     private val bucketName: String,
     private val collectionName: String,
     private val validateSignature: Boolean = false,
-    client: HttpClient = HttpURLConnectionHttpClient()
+    client: Client = HttpURLConnectionClient()
```

Comment:
Now with `concept-fetch` I'd stop using `HttpURLConnection` by default (and having the dependency) and let the app define the `Client` (`HttpURLConnectionClient` is not something we actually recommend using unless you need a very small APK size). Currently an app already needs to construct and pass in `KintoExperimentSource` - so asking for a `Client` here doesn't seem an issue.

Reply:
@pocmo OK, updated and changelog / README note added for breaking API.